build/installEdgeDriver.ps1

```diff
@@ -1,4 +1,7 @@
-$url = "https://msedgedriver.azureedge.net/119.0.2151.72/edgedriver_win64.zip" #Chrome Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
+# Get the installed version of Microsoft Edge
+$edgeVersion = (Get-Item "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe").VersionInfo.ProductVersion
```

Comment:
Can you error out with some message if this is null?

Reply:
added error checks